subsys/bluetooth/host/att.c

```diff
@@ -3863,6 +3863,8 @@ int bt_att_req_send(struct bt_conn *conn, struct bt_att_req *req)
 	__ASSERT_NO_MSG(conn);
 	__ASSERT_NO_MSG(req);
 
+	k_sched_lock();
+
```

Comment:
if 
```c
	att = att_get(conn);
	if (!att) {
		return -ENOTCONN;
	}
```

fails, don't we need to unlock the scheduler? 

E.g. we have similar code in GATT:

```c
	k_sched_lock();

	err = gatt_register(svc);
	if (err < 0) {
		k_sched_unlock();
		return err;
	}
```

Reply:
c sux. that's all I have to say :p